spring-cloud-sleuth-samples/spring-cloud-sleuth-sample-grpc/README.md

```diff
@@ -0,0 +1,33 @@
+# Spring Cloud Sleuth GRPC
+
+This project demonstrates how Spring Cloud Sleuth can be used to instrument gRPC calls.
+
+The auto-configuration for instrumenting gRPC requires the following two libraries to be on the classpath:
+
+- grpc-spring-boot-starter
+- brave-instrumentation-grpc
+
+## Running without Zipkin:
+
+1. Edit src/main/resources/application.yml and uncomment the "sample.zipkin.enable=false"
+2. Import this maven project into your favorite IDE (note, I have not setup m2e to automatically build the stubs from within Eclipse)
+3. Run the spring boot application.
+4. In a browser "http://localhost:8080/hi/fred"
+
+
+## Running with Zipkin:
+
+1. For conveinence, there is a docker-compose file in the project that can be used to launch zipkin locally. Otherwise, you will need to run zipkin separately.
```

Comment:
Typo

```suggestion
1. For convenience, there is a docker-compose file in the project that can be used to launch zipkin locally. Otherwise, you will need to run zipkin separately.
```